dags/bqetl_addons.py

```diff
@@ -52,7 +52,11 @@
         dataset_id="telemetry_derived",
         project_id="moz-fx-data-shared-prod",
         owner="jklukas@mozilla.com",
```

Comment:
best to remove all `jklukas` mentions here

Reply:
Everything his name touches... is yours now.

Reply:
yeah, starting to realise my mistake now 😓

Reply:
I think Jeff should be no more in this PR.